netbox_librenms_plugin/tests/test_sync_view_mismatch.py

```
@@ -318,3 +318,329 @@ def test_vc_pattern_no_match_leaves_name(self, mock_settings_qs, mock_hw):
+
+        # get_librenms_sync_device must be called unconditionally for VC members
+        mock_sync_device.assert_called_once_with(member_a, server_key="default")
+        # The lookup device should be member_b, not member_a
+        assert view._librenms_lookup_device == member_b
+        # get_librenms_id should be called on the sync device (member_b)
+        api.get_librenms_id.assert_called_once_with(member_b)
+
+    @patch("netbox_librenms_plugin.views.base.librenms_sync_view.render")
+    @patch("netbox_librenms_plugin.views.base.librenms_sync_view.get_object_or_404")
+    @patch("netbox_librenms_plugin.views.base.librenms_sync_view.get_librenms_sync_device")
+    def test_vc_member_with_legacy_id_delegates_to_sync_device(self, mock_sync_device, mock_get_object, mock_render):
```

Flake8 (7.3.0)

[error] redefinition of unused 'test_vc_member_with_legacy_id_delegates_to_sync_device' from line 405

(F811)